parsers/fortran.c

```diff
@@ -582,10 +582,11 @@ static void makeLabelTag (vString *const label)
 
 static lineType getLineType (void)
 {
-	vString *label = vStringNew ();
+	static vString *label = NULL;
```

Comment:
Yeah, but there are quite a few of them - it's basically every "static vString" and there are quite many of them in the code already. On the other hand this isn't a super-important commit, the Fortran parser is already quite fast even without this patch.